src/config/settings.py

```diff
@@ -156,3 +156,17 @@
 ACCOUNT_FORMS = {
     "signup": "src.apps.user.forms.UserSignupForm",
 }
+
+REDIS_CACHE_HOST = env("REDIS_CACHE_HOST", default="localhost")
+REDIS_CACHE_PORT = env("REDIS_CACHE_PORT", default="6379")
+REDIS_CACHE_DB = env("REDIS_CACHE_DB", default="0")
+
+CACHES = {
+    "default": {
```

Comment:
lets use DummyCache by default
please do not add redis as default cache, because you will be 146% confused by test results